src/common/components/build-history/index.js

```diff
@@ -1,20 +1,23 @@
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
+import sortBy from 'lodash/sortBy';
 
 import BuildRow from '../build-row';
 import { Message } from '../forms';
 
 const BuildHistory = (props) => {
-  const { repository, success } = props;
+  const { repository, success, builds } = props;
 
-  const builds = props.builds.map((build) => (
+  sortBy(builds, ['buildId']);
```

Comment:
The docs say that `sortBy` returns the sorted array, not that it mutates it in place, so shouldn't this be `const sortedBuilds = sortBy(builds, ['buildId'])` or something like that?  (If it did mutate it in place, I'd worry slightly about whether this was mutating `props`.)